warehouse/query-core/src/main/java/datawave/query/transformer/ExcerptTransform.java

```diff
@@ -319,51 +337,70 @@ private Set<Excerpt> getExcerpts(PhraseIndexes phraseIndexes) {
      * @return the excerpt
      */
     private String getExcerpt(String field, int start, int end, Range range, ArrayList<String> hitTermValues) {
+        if (start < 0) {
+            start = 0;
+        }
+        float origHalfSize = (float) (end - start) / 2;
+        int expandSize = 20;
         excerptIteratorOptions.put(TermFrequencyExcerptIterator.FIELD_NAME, field);
```

Comment:
Consider making `excerptIteratorOptions` a local variable, I don't think there's a good reason to make it a class field.

Reply:
done